src/sentry/static/sentry/app/views/settings/project/navigationConfiguration.tsx

```diff
@@ -117,11 +117,15 @@ export default function getConfiguration({project}: ConfigParams): NavigationSec
           path: `${pathPrefix}/plugins/`,
           title: t('Legacy Integrations'),
           description: t('View, enable, and disable all integrations for a project'),
+          id: 'legacy_integrations',
+          recordAnalytics: true,
         },
         ...plugins.map(plugin => ({
           path: `${pathPrefix}/plugins/${plugin.id}/`,
           title: plugin.name,
           show: ({access}) => access.has('project:write'),
+          id: 'plugin',
```

Comment:
is this supposed to be `plugin_details` ? confused if this is for the integration_type or view

Reply:
@MeredithAnya I wasn't actually planning on tieing this to the view (we aren't doing so for the integrations page) but that's actually a good suggestion as it might open up some avenues of analysis. I will make the change here and the one for integration in a different PR.